projects/sandbox/deploy/deploy/main.py

```diff
@@ -1,25 +1,59 @@
 import logging
 import time
+from dataclasses import asdict, dataclass
 from pathlib import Path
-from typing import List
+from typing import List, Literal
 
 import numpy as np
 import torch
 from deploy.dataloading import data_iterator
+from ligo.gracedb.rest import GraceDb
 
 from bbhnet.analysis.ledger.events import EventSet
 from bbhnet.architectures import ResNet
 from bbhnet.logging import configure_logger
 
+Gdb_server = Literal["playground", "test", "production"]
+
 
 def unfold(x, kernel_size: int, stride: int):
     # TODO: fill out
     return
 
 
-def submit_trigger():
-    # TODO: fill out
-    return
+def connect_to_gracedb(server: Gdb_server):
+    if server in ["playground", "test"]:
+        server = f"https://gracedb-{server}.ligo.org/api/"
+    elif server == "production":
+        server = "https://gracedb.ligo.org/api/"
+    else:
+        raise ValueError(f"Unknown server {server}")
+
+    gdb = GraceDb(service_url=server)
+    return gdb
+
+
+@dataclass
+class Trigger:
+    time: float
+    detection_statistic: float
+    far: float
+
+
+def submit_trigger(gdb: GraceDb, trigger: Trigger):
+    filename = "event.json"
+    filecontents = str(asdict(trigger))
+    # alternatively we can write a file to disk,
+    # pass that path to the filename argument,
+    # and set filecontents=None
+    response = gdb.createEvent(
+        group="CBC",
+        pipeline="BBHNet",
+        filename=filename,
+        search="BBHNet",
```

Comment:
This should just be "BBH"

Reply:
I don't think there is just a "BBH" group. See https://gracedb-test.ligo.org/documentation/models.html?highlight=group

Reply:
Right, the group is "CBC", but the search should be "BBH"

Reply:
Ah yeah I see.

Reply:
Good catch. I'll update this on my PR and then close this one, which has integrated the changes here